### PR TITLE
Fixed %Y length in strptime

### PR DESCRIPTION
**Pull request by misamplus**
Check this line in your version:
```py
jdatetime.datetime.strptime('13980101000000', '%Y%m%d%H%M%S')
```

### CODE REVIEW

**Comment by slashmili** [comment]
Thanks @misamplus 

Could you please add a test case?

**Comment by misamplus** [comment]
I added it as `test_strptime_bare`

File: t/test.py [hunk 1]
```diff
@@ -373,6 +373,14 @@ def test_strptime(self):
...
 
         self.assertEqual(True, dt1 == dt2)
+    
+    def test_strptime_bare(self):
+        date_string = "13630606121314"
+        date_format = "%Y%m%d%H%M%S"
+        dt1 = jdatetime.datetime.strptime(date_string, date_format)
+        dt2 = jdatetime.datetime(1363, 6, 6, 12, 13, 14)
+
+        self.assertEqual(True, dt1 == dt2)
```
Comment (farzadghanei) [inline t/test.py]: we can use `self.assertTrue` instead of `assertEqual(True, )`

Comment (misamplus) [reply t/test.py]: I did it.

**Comment by slashmili** [comment]
Thanks @misamplus 🎉 

It's pushed as [3.6.1](https://pypi.org/project/jdatetime/3.6.1/)

**Pull request merged**